.github/workflows/test.yml

```diff
@@ -6,6 +6,10 @@ on:
     branches:
       - master
 
+concurrency:
+  group: ${{ github.ref }}
```

Comment:
I think we _don't_ want to cancel in-flight workflows on `master`; instead building every commit. This allows us to more easily tell where a change that breaks CI came from.